openhtf/util/measurements.py

```diff
@@ -151,15 +151,15 @@ def WithArgs(self, **kwargs):
     new_meas.docstring = formatter(self.docstring)
     return new_meas
 
-  def __getattr__(self, attr):
+  def __getattr__(self, attr):  # pylint: disable=invalid-name
```

Comment:
I'd rather fix pylint to understand the builtin magic methods than to add these everywhere